_posts/2021-07-30-release-1.12.5.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: post
+title:  "Apache Flink 1.12.5 Released"
+date:   2021-07-30 00:00:00
+categories: news
+authors:
+- jingsong:
+  name: "Jingsong Lee"
+
+---
+
+The Apache Flink community released the next bugfix version of the Apache Flink 1.12 series.
+
+This release includes 80 fixes and minor improvements for Flink 1.12.4. The list below includes a detailed list of all fixes and improvements.
```

Comment:
`1.12.4` -> `1.12.5`

Reply:
`1.12.4` is correct.

Reply:
The latest issue number included in this release should be 79 instead of 80 after the update.